packages/coin-sol/README.md

```diff
@@ -0,0 +1,112 @@
+# CoolWallet Stellar (XLM) SDK
```

Comment:
Should update `readme.md` for `coin-sol`.

Reply:
@hank121314 I have recently update `readme.md` file and relative file need to change, please check again.